### PR TITLE
Feature/cloud configuration

### PR DESCRIPTION
**Pull request by mithie**
- adjusted application-properties to use proper tls settings
- added Helm deployment documentation in README.md
- added deployment shell script
- added status endpoint in RESTController
- adjusted DynamoDB config

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,17 +8,125 @@ Admin backend to create new rooms. Containing security logic.
...
+  [Lombok](https://www.baeldung.com/lombok-ide) installation instructions for your IDE.
+  
+- [Helm](https://helm.sh/docs/using_helm/#install-helm) needs to be installed and properly configured for your AWS account
 
+- You need to know how to execute Helm charts manually or from an existing repository. If you do not have a custom Helm chart repository you can copy
+  the whole content from [viper-aws-helm-service-chart](https://github.com/visual-perceptibility/viper-aws-helm-service-chart) into a `helm` subfolder of
+  your current project root folder. You then have to adjust the `helm install` command in the `deploy-service.sh` file and provide the dedicated path to your
+  local helm chart.
 
 ## Start Applikation
```
Comment (christodo) [inline README.md]: Typo: Application

File: README.md [hunk 2]
```diff
@@ -8,17 +8,125 @@ Admin backend to create new rooms. Containing security logic.
...
+    docker push 111122223333.dkr.ecr.eu-west-1.amazonaws.com/viper/viper-service-remote-collab-admin:latest
+    ```
+
+### Deploy the service with pre-built helm chart
+
+1. CD into the project's root folder
+
+2. Execute the `deploy-service.sh` shell script by typing in
+ 
+   `deploy-serfice.sh <KEYSTORE_PASSWORD> <AWS_ACCESS_KEY> <AWS_SECRET_KEY>`
```
Comment (christodo) [inline README.md]: Typo: deploy-service.sh

File: src/main/java/com/bmw/remotecollab/admin/dynamoDB/DynamoDBConfig.java [hunk 3]
```diff
@@ -26,12 +31,18 @@
     @Value("${amazon.aws.secretkey}")
     private String dynamoDBSecret;
 
+
+    @Value("${amazon.aws.region}")
+    private String region;
+
     @Bean
     public AmazonDynamoDB amazonDynamoDB() {
+        logger.info("STarting DynamoDB with endpoint: {}", this.dynamoDBEndpoint);
```
Comment (christodo) [inline src/main/java/com/bmw/remotecollab/admin/dynamoDB/DynamoDBConfig.java]: Typo: Starting

File: src/main/java/com/bmw/remotecollab/admin/dynamoDB/DynamoDBConfig.java [hunk 4]
```diff
@@ -26,12 +31,18 @@
...
         return AmazonDynamoDBClientBuilder
                 .standard()
-                .withEndpointConfiguration(new AwsClientBuilder.EndpointConfiguration(dynamoDBEndpoint, "local"))
+                .withEndpointConfiguration(new AwsClientBuilder.EndpointConfiguration(dynamoDBEndpoint, region))
                 .withCredentials(amazonAWSCredentialsProvider())
+                // .withRegion("eu-west-1")
```
Comment (christodo) [inline src/main/java/com/bmw/remotecollab/admin/dynamoDB/DynamoDBConfig.java]: Do we still need this?

File: src/main/resources/application.properties [hunk 5]
```diff
@@ -1,17 +1,22 @@
...
+#DynamoDB Properties
+#amazon.dynamodb.endpoint: http://dynamodb-session:8000/
+amazon.dynamodb.endpoint: http://dynamodb.eu-west-1.amazonaws.com/
+amazon.aws.region: eu-west-1
+amazon.aws.accesskey: ${AWS_ACCESS_KEY} 
+amazon.aws.secretkey: ${AWS_SECRET_KEY}
+
+#OpenVidu Properties
+####AWS####
+openvidu.url: https://openvidu.viper.bmw.cloud:4443/
```
Comment (christodo) [inline src/main/resources/application.properties]: This isn't the correct URL for a local instance. Should either be local docker container or secret must be a variable

File: src/main/resources/application-prod.properties [hunk 6]
```diff
@@ -0,0 +1,21 @@
+
+#### Tomcat ####
+server.port: 5000
```
Comment (christodo) [inline src/main/resources/application-prod.properties]: Shouldn't it be port 443?

File: README.md (hunk 1 above)
Comment (mithie) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (mithie) [reply README.md]: done

File: src/main/java/com/bmw/remotecollab/admin/dynamoDB/DynamoDBConfig.java (hunk 3 above)
Comment (mithie) [reply src/main/java/com/bmw/remotecollab/admin/dynamoDB/DynamoDBConfig.java]: done

File: src/main/java/com/bmw/remotecollab/admin/dynamoDB/DynamoDBConfig.java (hunk 4 above)
Comment (mithie) [reply src/main/java/com/bmw/remotecollab/admin/dynamoDB/DynamoDBConfig.java]: removed comment

File: src/main/resources/application-prod.properties (hunk 6 above)
Comment (mithie) [reply src/main/resources/application-prod.properties]: seems as if application-prod.properties is never used. removed it.

File: src/main/resources/application.properties (hunk 5 above)
Comment (mithie) [reply src/main/resources/application.properties]: we need a better strategy for different environments. for the moment I'd suggest to replace the url locally when needed

**Pull request merged**